denops/@denops-private/service.ts

```
@@ -192,6 +192,142 @@ function assertValidPluginName(name: string) {
   }
 }
 
+type PluginModule = {
+  main: Entrypoint;
+};
+
+class Plugin {
+  #denops: Denops;
+  #loadedWaiter: Promise<void>;
+  #unloadedWaiter?: Promise<void>;
+  #disposable: AsyncDisposable = voidAsyncDisposable;
```

Biome (1.9.4)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)